### ISSUE TITLE
install pg module manually

### ISSUE THREAD

**Issue by kissmygritts** [sequelize/sequelize-auto#109]
When running `sequelize-auto ...` I received this error.

```
/usr/local/lib/node_modules/sequelize-auto/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:27
      throw new Error('Please install \'' + (sequelize.config.dialectModulePath || 'pg') + '\' module manually');
      ^

Error: Please install 'pg' module manually
    at new ConnectionManager (/usr/local/lib/node_modules/sequelize-auto/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:27:13)
    at new PostgresDialect (/usr/local/lib/node_modules/sequelize-auto/node_modules/sequelize/lib/dialects/postgres/index.js:12:28)
    at new Sequelize (/usr/local/lib/node_modules/sequelize-auto/node_modules/sequelize/lib/sequelize.js:233:18)
    at new AutoSequelize (/usr/local/lib/node_modules/sequelize-auto/lib/index.js:13:24)
    at Object.<anonymous> (/usr/local/lib/node_modules/sequelize-auto/bin/sequelize-auto:65:12)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:420:7)
    at startup (bootstrap_node.js:139:9)
    at bootstrap_node.js:535:3
```

Similar to this issue https://github.com/sequelize/sequelize/issues/6907, however I used npm to install my dependencies.

node : v7.3.0
npm : v3.10.10

and dependencies from package.json
```
...
  "dependencies": {
    "express": "^4.14.0",
    "pg": "^6.1.2",
    "sequelize": "^3.28.0"
  }
...
```

Installing pg globally solved the problem.

**Comment by iamkirkbater** [comment]
I left a similar comment on #6907, if you install sequelize globally, and then try to reference the locally installed node_modules/pg package, it won't work.  You have to install both globally or run the sequelize command from the node modules folder, `./node_modules/.bin/sequelize [command]`.

**Comment by steveschmitt** [comment]
I think this is fixed in `sequelize-auto@0.5.0`
Please try it and close the issue if so.  Thanks!

**Comment by steveschmitt** [comment]
Fixed in `sequelize-auto@0.5.1`.

**Comment by mocarskijosh** [comment]
@steveschmitt this is broken again 

Had to install both pg and sequelize globally to get past it.

`    "sequelize": "^6.37.1",
    "sequelize-auto": "^0.6.7",
    "sequelize-typescript": "^2.1.6",
`
